sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -50,89 +50,27 @@
       <!-- Azure SDK -->
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core</artifactId>
-        <version>1.26.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-cosmos</artifactId>
+        <artifactId>azure-cosmos</artifactId> <!-- Spring Cloud Azure is tested with 4.28.0 -->
         <version>4.28.0</version> <!-- {x-version-update;com.azure:azure-cosmos;dependency} -->
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core-management</artifactId>
-        <version>1.5.3</version> <!-- {x-version-update;com.azure:azure-core-management;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-amqp</artifactId>
-        <version>2.4.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-data-appconfiguration</artifactId>
-        <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-identity</artifactId>
-        <version>1.4.6</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-messaging-eventhubs</artifactId>
-        <version>5.11.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
-        <version>1.12.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-messaging-servicebus</artifactId>
-        <version>7.7.0</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-security-keyvault-certificates</artifactId>
-        <version>4.2.8</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;dependency} -->
+        <artifactId>azure-sdk-bom</artifactId>
+        <version>1.2.0</version>
```

Comment:
do we need to add a version hint here?

```
<!-- {x-version-update
```

Reply:
there's no entry for azure-sdk-bom in [version_client.txt](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/version_client.txt)